### ISSUE TITLE
CoinMarketCap API v1 shutting down

### ISSUE THREAD

**Issue by sz4bi** [witnessmenow/arduino-coinmarketcap-api#4]
Dear Brian,

CoinMarketCap Public API Version 1 will be taken offline on November 30th, 2018. Please modify your application to use Version 2 prior to shutdown.

Thank you.

**Comment by Lbenza** [comment]
Thats would be interesting, how we change to V2 Api?

**Comment by sz4bi** [comment]
V2 is documented there [link](https://coinmarketcap.com/api/)

**Comment by koosthoe** [comment]
Any word if this application will be updated to API V2? I am attempting to do it myself but am having trouble

**Comment by witnessmenow** [comment]
Yeah I plan to update it over the next couple of weeks. I'll close out this
issue when I do so you should get notified

V1 doesn't get shutdown til November so there is still plenty of time

**Comment by simplest-circuits** [comment]
You should switch to Coinlib.io 
They have a free API to get the JSON.

Perhaps i find time to do it by myself.

**Comment by bobbyong** [comment]
You may also consider CoinGecko's free API:
https://www.coingecko.com/api/

**Comment by sz4bi** [comment]
Now v2 api is migrating to 'pro'. Basic free plan for personal use is available.

**Comment by Lbenza** [comment]
This one still works?

**Comment by witnessmenow** [comment]
I have not migrated to V2, and probably won't, the free plan is very
limiting

**Comment by Lbenza** [comment]
so no more crypto api :(

**Comment by bobbyong** [comment]
You can try our Simple API. Example:
https://api.coingecko.com/api/v3/simple/price?ids=bitcoin&vs_currencies=usd

**Comment by Lbenza** [comment]
Will try today bobby, and I will tell you.

**Comment by Lbenza** [comment]
I just try the api v1 and is still working

**Comment by tom71** [comment]
Hello,
now the CoinMarketCap API is down. I tried to use the free API with an API-Token. This can work, but I can't parse the response from the API. (BTW I have the same problem with api.coingecko.com)
I receive the response and want to put it in a String, but in the repsonse are a unknown letter at the beginning and a 0 at the end.

For example:
`1b
{"bitcoin":{"eur":5851.43}}
0`
Did you have a hint for me?

Thank you
